docs/source/clioptions.md

```diff
@@ -0,0 +1,35 @@
+# CLI options for constructor-generated installers
+
+## Shell-based installers for MacOS and Linux
+
+We have the following CLI options available for MacOS and Linux:
+
+- If the installer is running in a batch, `-i` is used to run it in an interactive mode. Otherwise, the installer runs in batch mode without manual intervention by `-b`. 
+- `f` returns no error is the prefix already exists.
+- `h` allows us to display this help message and exit.
+- You may use `-k` to keep the package cache after installation.
+- You can use `-p PREFIX` to install a prefix. Kindly note that the path should not contain any spaces.
+- If you wish to skip running `pre/post-link/install`scripts, you may use `-s`.
+- `-u` is used to update an existing installation.
+- Considering you have conda installed, `-t` can be used to run package tests after installation (it may install conda-build).
```

Comment:
I'd rather have the options listed as we usually find them in help messages:

- Hyphenated flags first, then description
- Order is usually alphabetic, but feel free to group them differently if it makes sense.

Reply:
Sure, I'll make those changes. I decided on the order based on how they were defined in the code. I'll try to put it alphabetically, if it doesn't work out, I'll keep the order the same until the next review.